.github/workflows/reusable-linters.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         files: >-
           .tox/.tmp/.mypy/python-3.12/cobertura.xml,
           .tox/.tmp/.mypy/python-3.10/cobertura.xml,
```

Comment:
We should probably update these two in a separate PR.

Reply:
Are you thinking 3.11 and 3.13?

Reply:
Yeah. I'm trying to have every other version (almost) where I can. This should cover all the code branches. Another strategy is to only have min+max, but I think it doesn't hurt to run more. Ideally, every version in the range, but that's probably too much, so I settled on every other.

Reply:
OTOH, we could do it in this PR too...

Reply:
Adjusted